Sources/TSCUtility/Platform.swift

```diff
@@ -26,16 +26,16 @@ public enum Platform: Equatable {
 
     /// Lazily checked current platform.
     public static var currentPlatform = Platform._findCurrentPlatform(localFileSystem)
-    /// Attempt to match `uname` with recognized platforms.
-    /// Additionally, try `ver` to recognize Windows.
+
     public static func _findCurrentPlatform(_ fs: FileSystem) -> Platform? {
-        guard let uname = try? Process.checkNonZeroExit(args: "uname").spm_chomp().lowercased() else {
-            if let ver = try? Process.checkNonZeroExit(args: "ver"),
-                ver.hasPrefix("Microsoft Windows") {
-                return .windows
-            }
-            return nil
-        }
+        /// Recognize Darwin and Windows at compile time.
+      #if os(Darwin)
```

Comment:
I don't think that is is a valid check - this should be `os(iOS) || os(macOS) || os(tvOS) || os(watchOS)`

Reply:
What about `#if canImport(Darwin)`?

Reply:
That might work, though it's an unusual use of `canImport`.  I am not sure where I got the idea that `#if os(Darwin)` was supported (probably from this thread: https://forums.swift.org/t/idea-support-if-os-darwin-as-shorthand-for-os-ios-os-osx-os-watchos-os-tvos) but I was clearly wrong about that.  Thanks for catching that @compnerd.  I'm really not fond of spelling out the Darwin platforms though, so maybe we do need to go back to calling `uname` in this case, unfortunately.

Reply:
@abertelrud you're welcome :)